Documentation/dev/debugging.md

```diff
@@ -169,3 +170,34 @@ kubectl -n $METERING_NAMESPACE port-forward hdfs-datanode-0 9864
 ```
 
 To check other datanodes, run the above command, replacing `hdfs-datanode-0` with the pod you want to view information on.
+
+## Metering Ansible Operator
+Metering uses the ansible operator to watch and reconcile resources in a cluster environment. When debugging a failed Metering install, it can be helpful to view the Ansible logs or status of your `MeteringConfig` custom resource.
+
+##### Accessing Ansible Logs
+There are a couple of ways of accessing the Ansible logs depending on how you installed the Metering resources.
+
+In a typical install, the Metering operator is deployed as a pod. In this case, we can simply check the logs of the `ansible` container within this pod:
+```
+kubectl -n $METERING_NAMESPACE logs $(oc get pods -l app=metering-operator -o name | cut -c 5-) -c ansible
```

Comment:
You can use `cut -d/ -f2` to use `/` as the delimiter, and select the 2nd field, which is the pod name.